central/cluster/datastore/datastore_impl.go

```diff
@@ -353,6 +354,10 @@ func (ds *datastoreImpl) addClusterNoLock(ctx context.Context, cluster *storage.
 	if cluster.GetName() == "" {
 		return "", errors.New("cannot add a cluster without name")
 	}
```

Comment:
No need for this since it moved into the validation function.

Reply:
Removed in [19c9c6a](https://github.com/stackrox/stackrox/pull/12295/commits/19c9c6a718dfc86f7f88e2b7d3e1ef9ec3efdf85)